portal-kernel/src/com/liferay/portal/kernel/servlet/BufferCacheServletResponse.java

```diff
@@ -352,6 +352,14 @@ public void setContentLength(int contentLength) {
 
 	}
 
+	@Override
+	public void setContentLengthLong(long contentLength) {
+
+		// Buffered response cannot accept content length because content post
+		// processing may cause length change
+
+	}
+
```

Comment:
@holatuwol I am surprised that the addition of the setContentLengthLong(long) method did not cause a Semantic Version MAJOR change since package com.liferay.portal.kernel.servlet is exported. You might need to add a `@ProviderType` annotation.